packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -550,6 +575,23 @@ contract Attestations is
     return (matches, new address[](totalAddresses));
   }
 
+  function getPossibleAttestationIssuers() public view returns (address[] memory, uint256) {
```

Comment:
Was this extraction necessary? It also contains an error as it makes it possible to add the same issuer more than once?

Reply:
Yeah, it's more efficient to not extract everything at start. Added a test to check that validator cannot be selected twice.